addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.js

```diff
@@ -21,7 +21,7 @@ export class AvatarCardPopover extends Component {
     }
 
     get fieldNames() {
-        return ["name", "email", "phone", "im_status"];
+        return ["name", "email", "phone", "im_status", "share"];
```

Comment:
Note: from discuss code we typically already know this information in store/models and this will make an extra/unnecessary RPC.
But I understand the need to fetch missing data in other situations.
It would be nice to reconcile the two when possible to avoid RPC when avoidable.

Reply:
I don't ever see the phone number in said store/models so is there really a way to avoid the orm call completely?